apps/sage_intacct/connector.py

```
@@ -1724,3 +1727,55 @@ def post_charge_card_transaction(
+        :param journal_entry_lineitems: JournalEntryLineitem objects
+        :return: created journal_entry
+        """
+        configuration = Configuration.objects.get(workspace_id=self.workspace_id)
+        try:
+            journal_entry_payload = construct_journal_entry_payload(
+                workspace_id=self.workspace_id,
+                journal_entry=journal_entry,
+                journal_entry_line_items=journal_entry_line_items
+            )
+            created_journal_entry = self.connection.journal_entries.post(journal_entry_payload)
+            return created_journal_entry
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)